nuget/lib/dependabot/nuget/file_parser/project_file_parser.rb

```diff
@@ -14,6 +14,7 @@
 module Dependabot
   module Nuget
     class FileParser
+      # rubocop:disable Metrics/ClassLength
```

Comment:
I'm not planning to leave this as-is, but I didn't want to spend a bunch of time and energy refactoring it smaller until I was confident that my fix works.

Reply:
Apparently I ended up keeping it under the limit, though I could swear it's longer now than when rubocop complained.